### ISSUE TITLE
MultiSelect component crashes on keyboard navigation when no enabled items left.

### ISSUE THREAD

**Issue by fullas** [palantir/blueprint#3612]
#### Environment

- __Package version(s)__: 
```
"@blueprintjs/core": "3.15.1",
"@blueprintjs/select": "3.8.0",
```
- __Browser and OS versions__: Chrome 75, Ubuntu 16.04

#### Steps to reproduce

1. Select all items in a `MultiSelect` component
1. Close input then open it again
1. Press down arrow key

#### Actual behaviour

Crashes

#### Expected behaviour

Nothing should happen when there are no more items left to select

#### Possible solution

Possibly related to #3388

**Comment by jacekjagiello** [comment]
@fullas I would like to start working on this, but I can't reproduce crash behavior. 
Maybe I'm doing something wrong?

![multi_select](https://user-images.githubusercontent.com/2925620/60343843-7a35fe80-99b5-11e9-851a-069445d68a52.gif)

**Comment by fullas** [comment]
Disable selected items in popover.

**Comment by jacekjagiello** [comment]
@fullas this should be fixed in #3627
